clients/js/test/collect.test.ts

```diff
@@ -69,10 +72,10 @@ test('it can add remove asset plugin with collect amount', async (t) => {
   const umi = await createUmi();
   const asset = await createAsset(umi, {
     plugins: [
-      pluginAuthorityPair({
+      {
         type: 'FreezeDelegate',
-        data: { frozen: false },
-      }),
+        frozen: false,
+      },
     ],
   });
 
```

Comment:
There's some tests like 'it can add asset plugin with collect amount', and 'it can add remove asset plugin with collect amount'.

wdyt about adding a test that is like 'can add an external plugin with collect amount'?

Reply:
Yeah that makes sense!